addons/analytic/views/analytic_account_views.xml

```diff
@@ -273,8 +273,8 @@
             <field name="arch" type="xml">
                 <pivot string="Analytic Entries" >
                     <field name="account_id" type="row"/>
-                    <field name="unit_amount" type="measure"/>
-                    <field name="amount" type="measure"/>
+                    <field name="unit_amount" type="measure" widget="float_time"/>
```

Comment:
I don't think this should be done on the generic report. Analytic lines are used to record amounts on several models, including expenses or invoice lines. A duration is expected only for timesheet.

Reply:
Yep. Removed that.